modules/cnv-importing-vm-wizard-web.adoc

```diff
@@ -0,0 +1,110 @@
+. Click *VMware Virtual Disk Development Kit (VDDK)*.
+. Click *Latest Releases* and select the latest VDDK release.
+. Click *Download SDKs* to download the VDDK archive file.
+. Save the VDDK archive file in an HTTP-accessible location.
+. Go to the namespace where you will import the virtual machine:
++
+[options="nowrap" subs="+quotes,verbatim"]
+----
+# oc project <namespace>
+----
+
+. Update the `<url>` parameter with the path of the VDDK archive file and enter
+the following command:
++
+[options="nowrap" subs="+quotes,verbatim"]
+----
+# cat <<EOF | oc apply -f -
+apiVersion: cdi.kubevirt.io/v1alpha1
+kind: DataVolume
+metadata:
+  name: "vddk-pvc"
+spec:
+  source:
+    http:
+      url: "<http://www.example.com/VDDK_archive_file>"
+  contentType: "archive"
+  pvc:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: "200Mi"
+EOF
+----
+
+.Procedure
+
+. Click *Workloads* -> *Virtual Machines* from the side menu.
+. Click *Create Virtual Machine* and select *Create with Wizard*.
+. Click *Provision Source* and select *Import*.
+. Click *Provider* and select *VMware*.
+. Click *vCenter Instance* and select *Connect to New Instance* or an existing vCenter instance.
++
+* If you select *Connect to New Instance*, fill in the following fields:
+
+** *vCenter Hostname*
+** *vCenter User Name*
+** *vCenter Password*
++
+Click the *Check* button and, optionally, select *Remember vCenter Credentials*. The wizard connects to the vCenter instance.
++
+* If you select an existing vCenter instance, the wizard connects to it using the saved credentials.
+
+. Click *VM to import* and select a virtual machine.
++
+The *Name* and *Operating System* are filled in automatically with the attributes of the selected virtual machine. The *Flavor* will be the flavor that matches the virtual machine's CPU and memory configuration most closely. You can change these fields.
+
+. Select a *Workload Profile*.
+. Click *Next* to go to the xref:cnv-networking-wizard-fields-web[*Networking*] screen.
+. Click *NETWORK CONFIGURATION* and select *Pod Networking*.
+. Select a *BINDING METHOD*.
+. Click the &#10003; button to save the update.
+. Click *Next* to go to the xref:cnv-storage-wizard-fields-web[*Storage*] screen.
+. Click a disk to edit its attributes.
+. Select a *Bootable Disk* from the drop-down list.
+. Click the &#10003; button to save the update.
+. Click *Create Virtual Machine >*. The *Result* screen displays the resources created for the virtual machine.
+. Click *Close*.
++
+The virtual machine, its namespace, and its state, are displayed in *Workloads* -> *Virtual Machines*. You can view the import progress by hovering your cursor over the state.
+
+[IMPORTANT]
+====
+Currently, when using local storage, only one virtual machine can be imported into a namespace. To import additional virtual machines into the same namespace, delete the conversion pod before importing:
```

Comment:
s/pod/Pod all places, like almost all [API objects](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#api-object-formatting).

Reply:
OK, but it looks strange.